integration-tests/README.md

```diff
@@ -68,7 +68,7 @@ The list of test groups can be found at
 
 ### Run a single test
 
-To run only a single test using Maven:
+To run only a single test using Maven, run the following command from `integration-tests` directory:
```

Comment:
This seems weird to me. 
When we do `mvn verify -P integration-tests -Dgroups=<test_group> -Dit.test=<test_name>` this its generally the directory of the root druid pom. What's the error you get running from the root dir ?

Reply:
Doing it from the root directory wasn't running the test, and was failing with:
```
[ERROR] Failed to execute goal org.jacoco:jacoco-maven-plugin:0.8.4:check (report) on project druid-azure-extensions: Coverage checks have not been met. See log for details. -> [Help 1]
```

I'm unable to simulate the above error now for some reason. Running the above command now from root directory isn't doing with any actual steps, and seems stuck at the following:
<img width="881" alt="image" src="https://github.com/apache/druid/assets/31135861/d3574e60-d637-4539-9ac6-5a2c042d5839">

But running the command from `integration-tests` directory starts doing the steps - building Docker image etc.

Amatya also had this same issue. @AmatyaAvadhanula can you confirm? Thanks!

Reply:
I have had issues running the command from the root directory as well.
Adding `-pl integrations-tests` to the command and running it from the root directory helped solve the problem.

Reply:
@cryptoe 

I tried running `mvn verify -P integration-tests -Dgroups=batch-index` from root directory, it didn't run any test and exited with the following error:
```
[ERROR] Failed to execute goal org.jacoco:jacoco-maven-plugin:0.8.7:check (report) on project druid-azure-extensions: Coverage checks have not been met. See log for details. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :druid-azure-extensions
```

Then tried running `mvn verify -P integration-tests -Dgroups=batch-index -Djacoco.skip=true` from the root directory, it ran a bunch of tests for `com.google.common.collect.testing.testers` (not sure why), and then exited with the following error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.2.5:test (default-test) on project simple-client-sslcontext: groups/excludedGroups require TestNG, JUnit48+ or JUnit 5 (a specific engine required on classpath) on project test classpath -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :simple-client-sslcontext
```

In comparison, running the command from `integration-tests` directory (or with `-pl integration-tests` flag from the root directory) started creating the certificates etc as per our integration-tests scripts, and then started executing our tests.